app-android/src/androidTest/kotlin/care/data4life/integration/app/crud/data/BaseAppDataCrudTest.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022 D4L data4life gGmbH - All rights reserved.
+ */
+
+package care.data4life.integration.app.crud.data
+
+import care.data4life.integration.app.crud.BaseCrudSdkTest
+import care.data4life.integration.app.data.wrapper.Result
+import care.data4life.integration.app.data.wrapper.Result.Failure
+import care.data4life.integration.app.data.wrapper.Result.Success
+import care.data4life.integration.app.data.wrapper.awaitCallback
+import care.data4life.sdk.SdkContract.CreationDateRange
+import care.data4life.sdk.SdkContract.UpdateDateTimeRange
+import care.data4life.sdk.call.CallContract.Record
+import care.data4life.sdk.call.DataRecord
+import care.data4life.sdk.data.DataResource
+import org.threeten.bp.LocalDate
+import org.threeten.bp.LocalDateTime
+
+abstract class BaseAppDataCrudTest : BaseCrudSdkTest<DataResource>() {
+
+    override suspend fun callCreate(items: List<DataResource>): List<Result<DataRecord<DataResource>>> {
+        val results: MutableList<Result<DataRecord<DataResource>>> = mutableListOf()
+        for (index in 0 until items.count()) {
```

Comment:
`repeat` is here more idiomatic I think

Reply:
Yes, but you loose the index information